identity/client/src/pages/mappings/modals/AddModal.tsx

```diff
@@ -62,27 +69,39 @@ const AddMappingModal: FC<UseModalProps> = ({ open, onClose, onSuccess }) => {
         placeholder={t("Enter mapping ID")}
         onChange={setMappingId}
         value={mappingId}
+        helperText={t("Enter a unique ID for this mapping")}
         autoFocus
       />
       <TextField
         label={t("Mapping name")}
         placeholder={t("Enter mapping name")}
         onChange={setMappingName}
         value={mappingName}
+        helperText={t("Enter a unique name for this mapping")}
         autoFocus
       />
-      <TextField
-        label={t("Claim Name")}
-        placeholder={t("Enter claim name")}
-        onChange={setClaimName}
-        value={claimName}
-      />
-      <TextField
-        label={t("Claim Value")}
-        placeholder={t("Enter claim value")}
-        onChange={setClaimValue}
-        value={claimValue}
-      />
+      <MappingRuleContainer>
+        <Stack gap={spacing05}>
+          <h3>Mapping Rule</h3>
+          <CustomStack orientation="horizontal">
+            <TextField
+              label={t("Claim Name")}
+              placeholder={t("Enter claim name")}
+              onChange={setClaimName}
+              value={claimName}
+              helperText={t("Enter a custom claim name")}
+            />
+            <EqualSignContainer>=</EqualSignContainer>
+            <TextField
+              label={t("Claim Value")}
+              placeholder={t("Enter claim value")}
+              onChange={setClaimValue}
+              value={claimValue}
+              helperText={t("Enter the value for the claim")}
```

Comment:
All uses of the translation function need keys (gotta double check the values on Figma as well, like "Create new mapping" title for this modal is actually just "Create mapping" in Figma.